test/cppsuite/configurations/config.txt

```diff
@@ -0,0 +1,19 @@
+collection_count=2
```

Comment:
Rename this file to config_poc_short.txt

Also add a comment line explaining what this config is and what it does in terms of the workload at the top of this file.